spec/models/github_issue_spec.rb

```diff
@@ -88,10 +88,12 @@
     end
   end
 
-  xit "saves HTML in .processed_html" do
+  it "saves HTML in .processed_html" do
```

Comment:
This was a skipped test previously, but now that I've "un-`approvals`"ed it, I think we can unskip it :) 

It seems to consistently pass for me, but if it becomes flaky again or if we need to skip it again down the road, I figured we can cross that bridge when we come to it.